sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/SymbolicExecution/ExplodedGraphTests.cs

```diff
@@ -990,6 +990,52 @@ public void ExplodedGraph_TupleExpressionsWithDiscardDeconstruct()
             context.WalkWithInstructions(3);
         }
 
+        [TestMethod]
+        [TestCategory("Symbolic execution")]
+        public void ExplodedGraph_DiscardDesignationAsFunctionArgument()
+        {
+            const string testInput = @"var result = obj.Read(out int _);";
+
+            var context = new ExplodedGraphContext(testInput);
+            var resultSymbol = context.GetSymbol("result");
+
+            context.ExplodedGraph.InstructionProcessed +=
+                (sender, args) =>
+                {
+                    var instruction = args.Instruction.ToString();
+
+                    switch (instruction)
+                    {
+                        case "obj":
+                            args.ProgramState.GetSymbolValue(resultSymbol).Should().BeNull();
+                            args.ProgramState.ExpressionStack.Peek().ToString().Should().Be("SV_2"); // obj => SV_2
```

Comment:
what is SV_2 , SV_2.Read?

Reply:
SV_2 is the "obj" identifier and "SV_2.Read" is "obj.Read". I've put some comments at the end of the line but since the line is quite long they are not easily visible on github.

Reply:
I had to remove them in the end since the symbolic value counter is static and we will get different values depending on the order of the tests.

Reply:
> I've put some comments at the end of the line
well... I thought we said we wouldn't do that on long lines :)